websocket.js

```diff
@@ -12,6 +12,7 @@ if (typeof process !== 'undefined' && process) {
     module.exports = globalThis.WebSocket
   } else {
     // Node.js or Electron browser process
+    // eslint-disable-next-line unicorn/no-typeof-undefined
```

Comment:
Is there a good reason for `typeof`? Why not just `globalThis.WebSocket === undefined`?

Reply:
I think the reason is If WebSocket is not defined (like in older browsers versions, server-side runtimes etc), directly accessing globalThis.WebSocket might throw a ReferenceError if WebSocket is not even declared.

Using typeof ensures the check does not break, as typeof safely returns 'undefined' instead of throwing an error.